white-paper/wp.tex

```diff
@@ -344,12 +344,27 @@ \subsubsection{Discovery}
     such as front-ends and back-ends receive these messages and update local cache of node locators.
     These approach dramatically improve lookup performance in local networks, e.g. when most of communications
     are performed in one local region, and all region nodes are connected via local networ.
-    This protocol supposed to be used with other discovery dechicues.
+    This protocol supposed to be used with other discovery techniques.
     For security reasons, in untrusted networks, the broadcast messages may be optionally signed with
     node private keys to be verified by the receiver using CA public certificate.
-    For optimization reasons, this protocol is resused by metadata exchange protocol for
+    For optimization reasons, this protocol is reused by metadata exchange protocol for
     repository hash table propagation (see \ref{sec:metadata}). \todo{lookup for researches}.
-  \item[Distributed DB] - network addresses could be propagated to the system using supplimentary distributed
+    There are 2 common ways to implement LDP: \href{http://bittorrent.org/beps/bep_0014.html}{bep 14} and \href{http://bittorrent.org/beps/bep_0026.html}{bep 26}.
+    Bep 14 is SSDP-like style and Bep 26 is zeroconf style LDP. To implement Bep 26 each host is required to run a zeroconf service discovery daemon.
+    There is a popular \href{https://github.com/grandcat/zeroconf}{go zeroconf implementation} that could be used.
+    BitTorrent uses following multicast groups: A) 239.192.152.143:6771 (org-local) and B) [ff15::efc0:988f]:6771 (site-local) for Bep 14 implementation.
+    They've chosen such IPs because 239.192.0.0/14 is defined to be the IPv4 Organization Local Scope,
+    and is the space from which an organization should allocate sub-
+    ranges when defining scopes for private use.
+    ff15::efc0:988f also comes from \href{http://tools.ietf.org/html/rfc4291#section-2.7}{IPv6 spec} and means:
+    \begin{itemize}
+      \item[--] FF == Multicast
+      \item[--] 1 == ‘Flags' - where 1 indicates a non-permanently-assigned ("transient” or "dynamically" assigned) multicast address.
+      \item[--] 5 == Site-Local scope
+      \item[--] efc0:988f - the hex representation of 239.192.152.143
+    \end{itemize}
+    We could easily implement BEP 14 as it described and take \href{https://github.com/transmission/transmission/blob/7f147c65fb07a6baed3d079703ff0a31d1b1ca4c/libtransmission/tr-lpd.c}{this implementation} as an example.
```

Comment:
@stepanvalyavskiy please use `\code{}` function to display IP addresses with monospaced font

Reply:
Done